test/python/pulse/test_experiment_configurations.py

```diff
@@ -17,10 +17,30 @@
 
 from qiskit.pulse.channels import DriveChannel, MeasureChannel, AcquireChannel
 from qiskit.pulse.exceptions import PulseError
-from qiskit.pulse import LoConfig
+from qiskit.pulse import LoConfig, LoRange
 from qiskit.test import QiskitTestCase
 
 
+class TestLoRange(QiskitTestCase):
+    """Test LO LoRange."""
+
+    def test_properties_includes_and_eq(self):
+        """Test creation of LoRange. Test upper/lower bounds and includes.
+        Test __eq__ for two same and different LoRange's.
+        """
+        lo_range_1 = LoRange(lower_bound=-0.1, upper_bound=+0.1)
+
+        self.assertEqual(lo_range_1.lower_bound, -0.1)
+        self.assertEqual(lo_range_1.upper_bound, +0.1)
+        self.assertTrue(lo_range_1.includes(0.0))
+
+        lo_range_2 = LoRange(upper_bound=+0.1, lower_bound=-0.1)
```

Comment:
Could you please reverse the order of these kwargs?

Reply:
Done! https://github.com/Qiskit/qiskit-terra/pull/2730/commits/ee3063dea8faa64e09aeeecba314dfba468e6a2d